blueocean-personalization/src/main/js/components/DashboardCards.jsx

```diff
@@ -49,6 +53,23 @@ export class DashboardCards extends Component {
             return null;
         }
 
+        const pausedCards = this.props.favorites
+          .filter(favorite => favorite.item.latestRun.state === 'PAUSED')
+          .map(favorite => {
+              const pipeline = favorite.item;
+
+              return (
+                <div key={favorite._links.self.href}>
+                    <PipelineCard
+                      router={this.props.router}
+                      runnable={pipeline}
+                      favorite
+                      onFavoriteToggle={(isFavorite) => this._onFavoriteToggle(isFavorite, favorite)}
+                    />
+                </div>
+              );
+        });
+
         const favoriteCards = this.props.favorites.map(favorite => {
```

Comment:
If the requirement is to show all paused pipelines irrespective of whether or not they are favorites, then I wonder if this should actually be part of dashboard rather than personalization and just contribute to the same extension point. Might need to refactor `PipelineCard` out into core-js and make the favorite star an optional icon.

If this is just for favorites, that's fine, but the `{favoritesCards}` list should probably filter out the `PAUSED` pipelines so the same one doesn't show twice (unless that's the requirement)